grails-app/views/layouts/nrm_bs4.gsp

```diff
@@ -77,8 +77,13 @@
             <div class="ml-md-auto text-right">
                 <g:if test="${fc.currentUserDisplayName()}">
                     <div class="greeting text-right">G'day <fc:currentUserDisplayName/></div>
+                    <g:if test="${expSoon}">
```

Comment:
Just checking ${expiryDate} here will work and remove the need for the extra flag